debian/rules

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/make -f
+#export DH_VERBOSE = 1
+
+
+%:
+	dh $@ --with systemd
+
+override_dh_auto_build:
+	make -j build/insomnia
+
+override_dh_auto_test:
+	echo "skip tests"
```

Comment:
why?

Reply:
requires lots of stuff to do. For example, generate mocks, which is tricky under debuild :)
At the same time we run tests in Travis before debuild